mezzanine/core/managers.py

```diff
@@ -171,13 +171,18 @@ def search(self, query, search_fields=None):
             queryset = queryset.filter(reduce(ior, optional))
         return queryset.distinct()
 
-    def _clone(self, *args, **kwargs):
+    def _clone(self):
         """
         Ensure attributes are copied to subsequent queries.
         """
-        for attr in ("_search_terms", "_search_fields", "_search_ordered"):
-            kwargs[attr] = getattr(self, attr)
-        return super(SearchableQuerySet, self)._clone(*args, **kwargs)
+        search_terms = self._search_terms
+        search_fields = self._search_fields
+        search_ordered = self._search_ordered
+        clone = super(SearchableQuerySet, self)._clone()
+        clone._search_terms = search_terms
```

Comment:
Can't these just be assigned directly with vars? eg `clone._search_terms = self._search_terms`

Reply:
Yep, they can. I've pushed an update.

Reply:
Excellent, thank you